src/mlpack/core/data/map_policies/missing_policy.hpp

```diff
@@ -85,7 +85,8 @@ class MissingPolicy
               MapType& maps,
               std::vector<Datatype>& /* types */)
   {
-    static_assert(std::numeric_limits<T>::has_quiet_NaN == true,
+    static_assert(
+        std::numeric_limits<T>::has_quiet_NaN == true,
```

Comment:
Here I also don't see a reason to put the condition on a second line, I think it would have been fine on the first.